### PR TITLE
Resolves 526, create form element component

### PR DESCRIPTION
**Pull request by ClassicSours**

### CODE REVIEW

File: src/app/app.module.ts [hunk 1]
```diff
@@ -13,11 +13,13 @@ import {ConfigService} from './config.service';
...
 import { ContributingComponent } from './contributing/contributing.component';
 import { HomeComponent } from './home/home.component';
+import {BossyFormElementComponent} from '../bossy-ui/components/form-element/form-element'
 
 const appRoutes: Routes = [
   { path: '', component: HomeComponent },
   { path: 'sandbox', component: SandboxComponent },
-  { path: 'contributing', component: ContributingComponent }
+  { path: 'contributing', component: ContributingComponent },
+  { path: 'element', component: BossyFormElementComponent }
```
Comment (tabersays) [inline src/app/app.module.ts]: This additional path likely should not be in the PR

File: src/bossy-ui/components/form-element/form-element.css [hunk 2]
```diff
@@ -0,0 +1 @@
+
```
Comment (tabersays) [inline src/bossy-ui/components/form-element/form-element.css]: Useless file remove

File: src/bossy-ui/components/form-element/form-element.enum.ts [hunk 3]
```diff
@@ -0,0 +1,13 @@
+export enum BossyFormElementEnum {
+  calendar,
+  dropdown,
+  // dropdownmenuitem,
```
Comment (tabersays) [inline src/bossy-ui/components/form-element/form-element.enum.ts]: These comment pose no use and should be removed

**Comment by glawson** [comment]
@ClassicSours - please update, see Taber's comments.

File: src/bossy-ui/components/form-element/form-element.enum.ts (hunk 3 above)
Comment (glawson) [reply src/bossy-ui/components/form-element/form-element.enum.ts]: Don't call it an enum - it's not.  Please add as a 'string constant' in the FormConstantService - #529

**Comment by ClassicSours** [comment]
Sorry, updated with https://github.com/buildcom/BossyUI/tree/form-element
Uses fork and pull method instead of pull request
removed unnescesary file (enum that did nothing) and added css to form-element.css
resolves ng lint issue and passes codeship
included config's for each element

**Pull request closed without merging**